ui/features/landing/landing.controller.js

```diff
@@ -181,7 +180,10 @@ export default class LandingController {
                   return diagonal({source: o, target: o});
               })
               .style("stroke", function(d){
-                  return axis ? color[type] : color[d.target.nodeType - 1];
+                  if(!axis){
```

Comment:
My personal style is to have a space after a conditional keyword, to distinguish it from a function invocation. I like:
```
if (foo) {
```